common/utils.go

```diff
@@ -14,15 +14,17 @@ import (
 	"net"
 	"net/http"
 	"os"
+	"strings"
 	"time"
 
 	"github.com/Azure/azure-container-networking/log"
 )
 
 const (
 	metadataURL           = "http://169.254.169.254/metadata/instance?api-version=2017-08-01&format=json"
-	httpConnectionTimeout = 10
-	headerTimeout         = 20
+	azCloudUrl            = "http://169.254.169.254/metadata/instance/compute/azEnvironment?api-version=2018-10-01&format=text"
+	httpConnectionTimeout = 7
```

Comment:
why changed to 7?

Reply:
i thought 10 is too long for a timeout. what say